index.html

```diff
@@ -0,0 +1,25 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body class="container">
+    <header class="toolbar">
+        <img src="./img/my-logo.png" alt="my logo" class="my-logo">
+        <button class="hamburger-menu"><img src="./img/hamburger.png" alt="Menu" id="hamburger-img"></button>
+    </header>
+    <section class="headline">
+        <h3>I'm Newton <br>Glad to see you!</h3>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <span>
+            <h4 class="social-media-title">LET'S CONNECT</h4>
+            <a href="https://www.linkedin.com/in/alhassan-o-83039a80/" target="_blank"><img src="./img/linkedin-icon.png" alt="linkedin Profile"></a>
+            <a href="https://github.com/flemton" target="_blank"><img src="./img/github-icon.png" alt="GitHub Profile"></a>
+            <a href="https://web.facebook.com/n3wtongh/?_rdc=1&_rdr"><img src="./img/facebook-icon.png" alt="Facebook Profile" width="18px" height="20px"></a>
+        </span>
```

Comment:
- [x] Please make sure that your social media links are displayed using list items. The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)
  - for example
  ```html
  <ul>
   <li>
     <a><img src="./images/Linkedin icon.png" alt="none"></a>
   </li>
  </ul>
  ```

Reply:
> @Owusu-Desmond Thank you very much, sir, for pointing me in the right direction, but I want to ask, should I put all the social media icons from the Figma into my project or only those I have accounts with. and also when I run stylelint I get an error that says unexpected missing generic font family.

Hello @flemton, keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉. For this reason, you should make a perfect replica of the Figma template (or as similar as you can). If you want, after finishing this project, you can customize it as you wish. For now, we need to make sure you know how to follow design guidelines.

Reply:
@Gopxfs oh okay I get get and thank you very much